CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ set (CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules/"
 # Find UNIX commands
 include (FindUnixCommands)
 find_package (Git)
-find_program (GS gs gswin64)
+find_program (GS NAMES gs gswin64 gswin32)
```

Comment:
Actually I think what is used on Windows are the gswin64c/gswin32c but I don't know why do we search for the ghost on Windows. The only module that uses it is _psconvert_, and it finds the executable via a registry inquire, which was a complicated thing to achieve because Windows64 have two registries one for 32 and another 64 and the Ghost will be in either one or the other depending on its bitage.

Reply:
You're right. Seems we even don't need to search gs on Linux. The variable `GS` is never used in the cmake configuration or the psconvert source code. Maybe we can remove the `find_program (GS ...)` line?

Reply:
Indeed, it looks we can drop it.